py3status/modules/xfce4_notifyd.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+Display and toggle the DND (do-not-disturb) status of xfce4-notifyd
+
+Toggles DND status on click
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 5)
+    dnd_off: status string when DND is off (default 'Off')
+    dnd_on: status string when DND is on (default 'On')
+    format: display format for this module (default 'DND: {status}')
+
+Format of status string placeholders:
+    {status} replaced with dnd_off when DND is off, dnd_on when it is on
+
+Color options:
+    color_good: DND off
+    color_bad: DND on
+
+Examples:
+```
+xfce4_notifyd {
+     dnd_off = "🙉"
+     dnd_on = "🙈"
+}
+```
+
+Requires:
+    dbus: python bindings for dbus
+
+@author Robert Ricci <robert.ricci@gmail.com>
+@license BSD
+"""
+
+import dbus
+
+bus_name = 'org.xfce.Xfconf'
+channel = 'xfce4-notifyd'
+interface_name = 'org.xfce.Xfconf'
+object_path = '/org/xfce/Xfconf'
+setting = '/do-not-disturb'
```

Comment:
Variables here should be all capitalized... or be in `post_config_hook` with `self.` appended.

Reply:
Will fix.